DocsAndHospitals/Program.cs

```diff
@@ -0,0 +1,58 @@
+﻿using DocsAndHospitals.Models;
+using DocsAndHospitals.Services;
+using DocsAndHospitals.UI;
+using DocsAndHospitals.Factories;
+
+namespace DocsAndHospitals;
+
+internal class Program
+{
+    static void Main()
+    {
+        Hospital[] hospitals = HospitalFactory.CreateMany();
+        MainMenu(hospitals);
+    }
+
+    private static void MainMenu(Hospital[] hospitals)
+    {
+        IOutput output = new ConsoleOutput();
+        IInput input = new ConsoleInput();
```

Comment:
may be we can remain inputs in console, and business logic will only accept needed data as method parameters